net/isc-dhcp/files/dhcpd.init

```diff
@@ -1,33 +1,20 @@
 #!/bin/sh /etc/rc.common
 
 START=65
+USE_PROCD=1
+PROG=/usr/sbin/dhcpd
 
 lease_file=/tmp/dhcpd.leases
 config_file=/etc/dhcpd.conf
-pid_file=/var/run/dhcpd.pid
 
-start() {
+start_service() {
 	if [ ! -e $lease_file ]; then
 		touch $lease_file
 	fi
 
-	/usr/sbin/dhcpd -q -cf $config_file -lf $lease_file -pf $pid_file $dhcp_ifs
-
-	if [ $? -ne 0 ]; then
-		return 1
-	fi
-}
-
-stop() {
-	if [ ! -e $pid_file ]; then
-		return 1
-	fi
-
-	kill -9 `cat $pid_file`
-
-	if [ $? -ne 0 ]; then
-		return 1
-	fi
-
-	rm $pid_file
+	procd_open_instance
+	procd_set_param stdout 1
+	procd_set_param stderr 1
+	procd_set_param command $PROG -f -cf $config_file -lf $lease_file $dhcp_ifs
+	procd_close_instance
 }
```

Comment:
Consider:
```
service_triggers() {
  # use soft reload to prevent continuous stop-start and cache flush
  procd_add_reload_trigger "dhcp"
  procd_add_raw_trigger "interface.*.up" 2000 /etc/init.d/dhcpd reload
}
```